src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  box-sizing: border-box;
+
+  background-color: white;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  padding: 32px 16px 16px;
+
+
+  &__photo {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto;
+  }
+
+  &__name {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    padding-top: 40px;
+  }
+
+  &__code {
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    padding-top: 4px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    padding-top: 16px;
+  }
+
+  &__reviews {
+    color: $main-accent;
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
it's a default value for this, remove it in all cases
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#formal_definition
```suggestion
```